tasks/sanity-inspect-image.yaml

```diff
@@ -49,3 +51,7 @@ spec:
       echo "The base image is $BASE_IMAGE, get its manifest now"
       skopeo inspect --no-tags docker://$BASE_IMAGE  > $BASE_IMAGE_INSPECT || true
       echo "$BASE_IMAGE" | tee $(results.BASE_IMAGE.path)
+
+      BASE_IMAGE_INSPECT_RAW=$(skopeo inspect --no-tags --raw docker://$BASE_IMAGE)
+      BASE_IMAGE_REPOSITORY="$(echo $BASE_IMAGE_INSPECT_RAW | jq -r ".annotations.\"org.opencontainers.image.source\"")"
```

Comment:
According to https://github.com/opencontainers/image-spec/blob/main/annotations.md, this `org.opencontainers.image.source` is `URL to get source code for building the image`, not a image repo.